stream-compositions/services/payment-order-composition-service/src/main/resources/application-local.yml

```diff
@@ -0,0 +1,76 @@
+server.port: 9005
+
+spring:
+  activemq:
+    broker-url: tcp://localhost:61616
+  zipkin:
+    enabled: false
+  security:
+    oauth2:
+      client:
+        registration:
+          dbs:
+            authorization-grant-type: client_credentials
+            client-id: bb-client
+            client-secret: bb-secret
+            client-authentication-method: post
+        provider:
+          dbs:
+            token-uri: http://localhost:8181/oidc-token-converter-service/oauth/token
+  main:
+    allow-bean-definition-overriding: true
+  codec:
+    max-in-memory-size: 10MB
+sso:
+  jwt:
+    internal:
+      type: signed
+      signature:
+        algorithm: HS256
+        key:
+          type: ENV
```

Comment:
For a local dev env you can make your life easier by:
```
sso.jwt.internal.signature.key:
  type: VALUE
  value: JWTSecretKeyDontUseInProduction!
```

Reply:
Nice! I updated this.